src/openapi/transport/batch.js

```diff
@@ -36,9 +36,10 @@ function emptyQueueIntoServiceGroups() {
 
 function batchCallFailure(callList, batchResponse) {
     const isAuthFailure = batchResponse && batchResponse.status === 401;
+    const isNetworkError =
+        !batchResponse || batchResponse.isNetworkError || !batchResponse.status;

```

Comment:
it is still a mystery as to why I would need to guard status here. pending any good ideas.

Reply:
@lukeapage  Do we still need this one with auth fix?